examples/IRrecvDumpV2/IRrecvDumpV2.ino

```diff
@@ -62,6 +62,11 @@ void  encoding (decode_results *results)
 //
 void  dumpInfo (decode_results *results)
 {
+  if (results->overflow) {
+    Serial.println("IR code too long. Edit IRremoteInt.h and increase RAWBUF");
```

Comment:
Suggestion/nitpick:
Probably should add how many elements we've received to the message.

And maybe move it to after displaying the decoding, and make it clearer that something is not right.
e.g. "WARNING: IR code too long. Could not decode properly. Edit ..."
That would make the output a little more useful, IMHO, but your approach is equally as valid a way to handle it.